Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,309 @@
+                                    Console.WriteLine("1. Total de Ventas");
+                                    Console.WriteLine("2. Clientes mayores a 55 años de edad");
+                                    Console.WriteLine("3. Clientes que cumplen años en el mes");
+                                    Console.WriteLine("4. Productos con stock bajo.");
+                                    Console.WriteLine("5. Regresar.");
+                                    Console.WriteLine("Seleccione una opcion");
+                                    boolAnswer = Int32.TryParse(Console.ReadLine(),out answer);
+                                }   while (boolAnswer == false);                              
+                                opcion = answer;
+                                switch (opcion)
+                                {
+                                    case 1: ReporteVentas(arraynuevaVenta);break; 
+                                            
+                                    case 2: ReporteClientes(arrayclienteNuevo);break;
+                                            
+                                    case 3: ReporteClientesEdad(arrayclienteNuevo);break; 
+                                            
+                                    case 4: ReporteProductos(arrayproductoNuevo);break; 
+                                            
+                                    case 5: break;
+                                }
+                            }
+                        }
+                        break;
+                }
+            }
+            Console.WriteLine("Gracias por su visita");
+        }
+        public static void ReporteVentas(List<Venta> arraynuevaVenta)
+        {
+            bool p= true;
+            Console.WriteLine("Total de Ventas del dia :"+ arraynuevaVenta.Count);
+            if(arraynuevaVenta.Count==0){
+                Console.WriteLine("NO HAY VENTAS REGISTRADAS");
+                p = false; 
+            }
+            else
+                foreach (Venta venta in arraynuevaVenta)
+                {
+                        if (p)
+                        {                            
+                            Console.WriteLine("--------------");
+                            Console.WriteLine("-Codigo del cliente :" + $"{venta.clienteId},  Codigo del producto: {venta.productoId}, Descripcion de la venta : {venta.description}");                               
+                        }
+                }
+        }
+        public static void ReporteClientes(List<Cliente> arrayclienteNuevo)
+        {
+            bool p= true;
+            Console.WriteLine("Clientes mayores a 55 años:");
+
+            if(arrayclienteNuevo.Count==0){
+                Console.WriteLine("NO HAY CLIENTES REGISTRADOS");
+            }
+            else
+                foreach (Cliente clients in arrayclienteNuevo)
+                {
+                        if (clients.age > 55)

```

Comment:
a property must to start with uppercase(pascal case)

Reply:
ready